### PR TITLE
Nndocs

### PR DESCRIPTION
**Pull request by fwilliams**
Also nuke modules we don't need/use

### CODE REVIEW

File: fvdb/nn/modules.py [hunk 1]
```diff
@@ -214,11 +274,38 @@ def reset_parameters(self) -> None:
...
     ) -> JaggedTensor:
+        """
+        Apply the sparse convolution to the input :class:`JaggedTensor` using the provided :class:`ConvolutionPlan`.
+
+        Args:
+            data (JaggedTensor): Input features to be convolved.
+            plan (ConvolutionPlan): The convolution plan defining the mapping between input and output grids.
+
+        Returns:
+            JaggedTensor: The result of the sparse convolution.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Should do the returns style like the others:

result (JaggedTensor): The result of the sparse convolution.

File: fvdb/nn/modules.py [hunk 2]
```diff
@@ -235,11 +322,38 @@ def forward(
...
     ) -> JaggedTensor:
+        """
+        Apply the sparse transposed convolution to the input :class:`JaggedTensor` using the provided :class:`ConvolutionPlan`.
+
+        Args:
+            data (JaggedTensor): Input features to be convolved.
+            plan (ConvolutionPlan): The convolution plan defining the mapping between input and output grids.
+
+        Returns:
+            JaggedTensor: The result of the sparse transposed convolution.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Should do the returns style like the others:

result (JaggedTensor): The result of the sparse transposed convolution.

File: fvdb/nn/modules.py [hunk 3]
```diff
@@ -214,11 +274,38 @@ def reset_parameters(self) -> None:
...
 class SparseConv3d(_SparseConv3dBase):
+    """
+    A sparse 3D convolution module that operates on :class:`JaggedTensor` inputs
+    according to a provided :class:`ConvolutionPlan`.
+
+    A :class:`ConvolutionPlan` defines the mapping of a sparse convolution operation
+    between data associated with an input :class:`fvdb.GridBatch` and an output
+    :class:`fvdb.GridBatch`. This allows for efficient sparse convolution operations
+    without explicitly constructing dense tensors.
+
```
Comment (swahtz) [inline fvdb/nn/modules.py:316]: Perhaps useful adding a `seealso` to Transposed Conv?

File: fvdb/nn/modules.py [hunk 4]
```diff
@@ -254,16 +368,32 @@ def forward(
...
     def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
+        """
+        Apply Group Normalization to the input :class:`JaggedTensor` using the provided :class:`GridBatch`.
+
+        Args:
+            data (JaggedTensor): Input features to be normalized.
+            grid (GridBatch): The grid batch corresponding to ``data``.
+
+        Returns:
+            JaggedTensor: The result of the group normalization.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Output arg naming

normalized_data (JaggedTensor): The result of the group norm

File: fvdb/nn/modules.py [hunk 5]
```diff
@@ -284,44 +414,78 @@ def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
...
     def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
+        """
+        Apply Batch Normalization to the input :class:`JaggedTensor` using the provided :class:`GridBatch`.
+
+        Args:
+            data (JaggedTensor): Input features to be normalized.
+            grid (GridBatch): The grid batch corresponding to ``data``.
+
+        Returns:
+            JaggedTensor: The result of the batch normalization.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Should do the returns style like the others:

normalized_data (JaggedTensor): The result of the batch normalization.

File: fvdb/nn/modules.py [hunk 6]
```diff
@@ -284,44 +414,78 @@ def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
...
+        Apply Synchronized Batch Normalization to the input :class:`JaggedTensor` using the provided :class:`GridBatch`.
 
         Args:
-            input: input JaggedTensor/GridBatch.
+            data (JaggedTensor): Input features to be normalized.
+            grid (GridBatch): The grid batch corresponding to ``data``.
 
         Returns:
-            Output JaggedTensor/GridBatch with batch norm applied to the feature dimension, across all ranks.
+            JaggedTensor: The result of the synchronized batch normalization.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Same output arg styling comment as above

File: fvdb/nn/modules.py [hunk 7]
```diff
@@ -330,17 +494,16 @@ def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
...
     @classmethod
     def convert_sync_batchnorm(cls, module: nn.Module, process_group: Any = None) -> nn.Module:
-        r"""Helper function to convert
-        :attr:`fvdb.nn.BatchNorm` layer in the model to
-        :attr:`fvdb.nn.SyncBatchNorm` layer.
+        """
+        Helper function to convert :attr:`fvdb.nn.BatchNorm` layer in the model to :attr:`fvdb.nn.SyncBatchNorm` layer.
 
         Args:
             module: Module for which all :attr:`fvdb.nn.BatchNorm` layers will be converted to
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Args missing types 

module(torch.nn.Module) …

File: fvdb/nn/modules.py [hunk 8]
```diff
@@ -330,17 +494,16 @@ def forward(self, data: JaggedTensor, grid: GridBatch) -> JaggedTensor:
...
                 :attr:`fvdb.nn.SyncBatchNorm` layers.
             process_group: process group to scope synchronization, default is the whole world.
 
         Returns:
-            The original module with the converted :attr:`fvdb.nn.SyncBatchNorm` layers.
+            torch.nn.Module:  The original module with the converted :attr:`fvdb.nn.SyncBatchNorm` layers.
```
Comment (swahtz) [inline fvdb/nn/modules.py]: Output arg naming for consistency:

sync_norm(torch.nn.Module): …
